strings/conf_to_src.cc

```diff
@@ -289,7 +289,6 @@ int main(int argc, char **argv [[maybe_unused]]) {
     exit(EXIT_FAILURE);
   }
 
-  memset(&ncs, 0, sizeof(ncs));
```

Comment:
should also change CHARSET_INFO to explicitly set everything to 0 in field initialization?

Reply:
Yeah, the current patch is incorrect due to doing default-initialization leaving fields uninitialized, not zero-initialized.